### PR TITLE
Resolve Issue #28

### PR DESCRIPTION
**Pull request by achang28**
For resolving https://github.com/acthp/ucsc-xena-client/issues/28, and reverting an html element back to previous.

### CODE REVIEW

File: js/Tooltip.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ var Tooltip = React.createClass({
 
 		var rowsOut = _.map(rows, (row, i) => (
 			<tr key={i}>
-				{row.map(([type, ...args], i) => element[type](i, ...args))}
+				{row ? row.map(([type, ...args], i) => element[type](i, ...args)) : element['value'][row]}
```
Comment (acthp) [inline js/Tooltip.js]: This is unlikely to be correct: if we have a list of rows, it shouldn't contain things which aren't rows. The bug is probably up the stack: either the value of the row is wrong, or there is no row.

Comment (achang28) [inline js/Tooltip.js]: Makes sense.  Going upstream...

File: js/plotDenseMatrix.js [hunk 2]
```diff
@@ -127,7 +127,7 @@ function bounded(min, max, x) {
 	return x < min ? min : (x > max ? max : x);
 }
 
-var nn = (...args) => args.filter(x => x != null); // not null
+var nn = (...args) => args.filter(x => (x != null) && (x != false)); // not null, nor boolean false
```
Comment (acthp) [inline js/plotDenseMatrix.js]: ```!=``` should be avoided, except when comparing to null, due to the oddities of js type coercion. I've updated the eslint rules to flag these. Should be ```!==```, instead. I believe we can drop the compare against null, and only check for false. Also, looks like this needs a rebase.

**Comment by achang28** [comment]
Oddly, I just tried return the original param, and it "seems" to be working:
var nn = (...args) => args.filter(x => x);

Within the callback provided to .filter( ), I tried a variety of return tests such as an empty array and "untruthy" values like undefined and null.  All appeared to work.

**Comment by acthp** [comment]
Not so odd... x => x  is same as  x => x !== false  when x is false.

That should work.

**Comment by achang28** [comment]
Please reject this PR.  Submitting different, cleaner, PR for this change.

**Pull request closed without merging**